tensorflow/lite/delegates/gpu/common/model_builder.cc

```diff
@@ -1412,21 +1413,32 @@ class MulOperationParser : public TFLiteOperationParser {
         input_tensor0 = 1;
         input_tensor1 = 0;
       }
-      return ParseApplyMask(node, input_tensor0, input_tensor1, graph, reader);
+      RETURN_IF_ERROR(ParseApplyMask(node, input_tensor0, input_tensor1, graph,
+                                     reader));
+    }
```

Comment:
can you merge L.1418 & L.1419?  Otherwise, everything looks fine.

Reply:
Thanks